src/Console/CheckoutCommand.php

```diff
@@ -162,21 +164,43 @@ private function buildVersionMap(array $config, OutputInterface $output)
         $map = [];
         $versions = array_keys($config['versions']);
 
-        foreach ($versions as $version) {
-            $modulesUrl = "https://raw.githubusercontent.com/silverstripe/supported-modules/$version/modules.json";
```

Comment:
It just occurred to me that the fetching the data from raw.githubusercontent.com is simply better than using `MetaData::getAllRepositoryMetaData()`.

Reason being that if we update silverstripe/supported-modules in the future then changes will be automatically picked up without needing to run composer update + deploy on all the things which would be super annoying and definitely adds to the maintainence burden

Sorry :-/